sw/device/silicon_creator/rom/rom_start.S

```diff
@@ -302,6 +302,40 @@ LABEL_FOR_TEST(kRomStartWatchdogEnabled)
   // The following sequence enables the minimum level of entropy required to
   // initialize memory scrambling, as well as the entropy distribution network.
   li a0, TOP_EARLGREY_ENTROPY_SRC_BASE_ADDR
+  li a1, (TOP_EARLGREY_OTP_CTRL_CORE_BASE_ADDR + \
+            OTP_CTRL_SW_CFG_WINDOW_REG_OFFSET)
+
+  // Load entropy src health check settings from OTP.
```

Comment:
nit: We can consider using `crt_section_copy` if the registers are consecutive. We can then confirm that everything is as expected in rom.c. I think we should do it regardless to harden this configuration.

Reply:
Done. I added a second commit implementing the check.